rules/windows/builtin/system/microsoft_windows_IphIpsvc/win_system_possible_ipv6_dns_takeover.yml

```
@@ -0,0 +1,29 @@
+title: Possible IPV6 DNS Takeover
+id: d22df9cd-2aee-4089-93c7-9dc4eae77f2c
+status: test
```

```suggestion
status: experimental
```
HI,
new rule are experimental on the repository.
You can found more details here https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-rule-convention.md